src/test/java/org/reactome/release/BioModelsUtilitiesTester.java

```diff
@@ -6,21 +6,16 @@
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.Mockito;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
 
 import java.util.ArrayList;
 import java.util.Collection;
+import org.mockito.junit.MockitoJUnitRunner;
 
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.*;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({BioModelsUtilities.class})
-@PowerMockIgnore({"org.apache.logging.log4j.*", "javax.management.*", "javax.script.*",
-        "javax.xml.*", "com.sun.org.apache.xerces.*", "org.xml.sax.*", "com.sun.xml.*", "org.w3c.dom.*"})
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
These changes have all been tested, right?

Reply:
Yup, I ran the tests locally and they all still passed with the changes.